helm/values.yaml

```
@@ -11,7 +11,7 @@ coder:
     repo: "ghcr.io/coder/coder"
     # coder.image.tag -- The tag of the image, defaults to {{.Chart.AppVersion}}
     # if not set.
-    tag: ""
+    tag: "latest"
```

We shouldn't do this, otherwise the version won't be respected.
```suggestion
    tag: ""
```